types/assert-equal-jsx/assert-equal-jsx-tests.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {assertEqualJSX} from 'assert-equal-jsx';
```

Comment:
Thanks Andy, I'm looking into that now. I'm still relatively new to typescript.

I was able to get intellisense responses from inside vscode and they looked correct, what should I be doing instead?